pom.xml

```diff
@@ -456,7 +456,7 @@
         </repository>
     </repositories>
     <properties>
-        <swagger-parser-version>1.0.9-SNAPSHOT</swagger-parser-version>
+        <swagger-parser-version>1.0.9</swagger-parser-version>
```

Comment:
@wing328 There appears to be a problem with one of the dependencies, even with a fresh, unmodified clone. I filed a bug here: https://github.com/swagger-api/swagger-codegen/issues/1055